src/remote.rs

```diff
@@ -288,6 +288,25 @@ impl<'repo> Remote<'repo> {
         Ok(())
     }
 
+    /// like `fetch` but passing NULL for `refspecs` array effectively fetching all
+    pub fn fetch_all(
```

Comment:
I feel like `all` may be a bit misleading, as it fetches the default remote refspecs, which may not be everything.

Is there a particular reason that using `fetch` with an empty refspec list isn't sufficient to handle this use case?